.travis.yml

```diff
@@ -6,9 +6,9 @@ before_install:
   - scripts/install_toxiproxy.sh
 
 rvm:
-  - '2.4'
```

Comment:
Ruby 2.4 is still in security maintenance. Do we want to remove it from this open-source project?
https://www.ruby-lang.org/en/downloads/branches/

Reply:
Added it back!